lib/ansible/config/base.yml

```diff
@@ -1399,10 +1399,9 @@ DUPLICATE_YAML_DICT_KEY:
   ini:
   - {key: duplicate_dict_key, section: defaults}
   type: string
-  choices: &basic_error2
-    error: issue a 'fatal' error and stop the play
+  choices:
+    <<: *basic_error
     warn: issue a warning but continue
```

Comment:
Should we change the text for `warn` to say something else? Maybe that it is an alias of `warning` and will be removed at some point? Maybe @bcoca has some ideas on how to handle this.